specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4646,6 +4656,16 @@
         }
       }
     },
+	"AzureFirstPartyManagedCertificateParameters": {
+      "description": "Azure FirstParty Managed Certificate used for https",
```

Comment:
Can we make it a little more descriptive, like "Certificate managed by Azure First Party provider, used for https"